src/System.ComponentModel.Primitives/src/System/ComponentModel/InvalidEnumArgumentException.cs

```diff
@@ -49,8 +49,12 @@ public InvalidEnumArgumentException(string argumentName, int invalidValue, Type
             : base(SR.Format(SR.InvalidEnumArgument,
                                 argumentName,
                                 invalidValue.ToString(CultureInfo.CurrentCulture),
```

Comment:
nit: I don't think we actually need this `ToString` as `SR.Format` does this for us

Reply:
Right. Would you mind cleaning that up as part of this PR?